fhir-notification/src/main/java/com/ibm/fhir/notification/FHIRNotificationService.java

```diff
@@ -207,6 +208,12 @@ private FHIRNotificationEvent buildNotificationEvent(String operation, FHIRPersi
             event.setResourceId(resource.getId());
             event.setResource(resource);
 
+            // Adds the tenant id and datastore id
+            String tenantId = FHIRRequestContext.get().getTenantId();
+            String dsId = FHIRRequestContext.get().getDataStoreId();
+            event.setDatasourceId(dsId);
+            event.setTenantId(tenantId);
```

Comment:
I wonder if we should avoid setting these when the tenant and datastore are default/default, which would be typical for a single-tenant deployment.

Reply:
I think we should include them as a matter of practice.  It'll allow someone to recreate a request and retrieve the full data and use it purposefully (and not by accident)